### PR TITLE
Pull request to add new layout Persian

### PR DESCRIPTION
**Pull request by RZHSSNZDH**

### CODE REVIEW

File: res/xml/persian.xml
Comment (Julow) [inline res/xml/persian.xml:23]: This is the right hand side of the keyboard. Perhaps `key2` is easier to reach ?

**Review by Julow — APPROVED** [review]
The layout looks good and thanks for the correction.

I'll leave the PR open for a bit of time to allow you to make improvements. It will be merged before the next release.
If you haven't had time to build the app locally, you can find the debug build here: https://github.com/Julow/Unexpected-Keyboard/actions/runs/4749949887

You include some ASCII symbols but not all, could more of them be useful ? For example, you have `*` but not `+-=`.

File: res/xml/persian.xml
Comment (RZHSSNZDH) [reply res/xml/persian.xml:23]: In my opinion `key2` is suitable for it. And also for math operators I think this is enough. If not we can change it later!

**Comment by RZHSSNZDH** [comment]
Sorry why can't I push to origin? I get error below:
`ERROR: Permission to Julow/Unexpected-Keyboard.git denied to RZHSSNZDH.`
`fatal: Could not read from remote repository.`

**Comment by Julow** [comment]
You can only push to your fork: https://github.com/RZHSSNZDH/Unexpected-Keyboard
That will update the PR automatically.

**Comment by Julow** [comment]
Did you mean to push something ? There's nothing new since 2 days ago.

I meant to leave this PR open until before the next release, in case you want to change the layout easily. If you prefer I can merge it now ?

**Comment by RZHSSNZDH** [comment]
So do not merge right now. I will take a look to the layout again.

**Comment by RZHSSNZDH** [comment]
I added math operators and changed `!`. You can merge.

**Review by Julow — APPROVED** [review]
Alright, I'll merge now.

Thanks for the translation!

**Pull request merged**
